generators/client/templates/angular/src/main/webapp/app/home/home.component.html.ejs

```diff
@@ -22,43 +22,44 @@
   </div>
 
   <div class="col-md-9">
-    <h1 class="display-4"><span <%= jhiPrefix %>Translate="home.title">Welcome, <%= backendName %> Hipster!</span> (<%= humanizedBaseName %>)</h1>
+    <h1 class="display-4"><span <%= jhiPrefix %>Translate="home.title"><%- this._getClientTranslation('home.title') %></span> (<%= humanizedBaseName %>)</h1>
 
-    <p class="lead" <%= jhiPrefix %>Translate="home.subtitle">This is your homepage</p>
+    <p class="lead" <%= jhiPrefix %>Translate="home.subtitle"><%- this._getClientTranslation('home.subtitle') %></p>
 
     <div [ngSwitch]="account !== null">
       <div class="alert alert-success" *ngSwitchCase="true">
+        <!-- TODO: 15964 Translate values -->
         <span id="home-logged-message" *ngIf="account" <%= jhiPrefix %>Translate="home.logged.message"
           [translateValues]="{ username: account.login }">You are logged in as user "{{ account.login }}".</span>
```

Comment:
With `_getClientTranslation` change, this will replace `{{ username }}` with `{{ account.login }}`
```suggestion
          [translateValues]="{ username: account.login }"><%- this._getClientTranslation('home.logged.message', { username: '{{ account.login }}' }) %></span>
```

Reply:
I will update all the places where translate values are used.

Reply:
I replaced everything that I understood how, only left the following dynamic value, because I'm not sure how to replace it correctly: `[translateValues]="{ id: <%= entityInstance %>.<%= primaryKey.name %> }"`

Also I'm not entirely sure if I replaced the values that are resolved by the generator correctly, for example for `[translateValues]="{ pattern: '<%= fieldNameHumanized %>' }"` I just used `{ pattern: fieldNameHumanized }` and I can't test that it works because entity-client generator does not have support for this method yet. 
So please let me know if this is wrong and I will fix it.